### DISCUSSION TITLE
Change yolov5 inference image size using sahi predict

### DISCUSSION THREAD

**Discussion by PogChamper** [obss/sahi#928] [Q&A]
Hello!
In yolov5 detect.py or val.py I can use flag --imgsz 640/1280/1536/any size for change inference size. Is it possible to do the same thing when I use sahi predict?
I haven't find questions like this in the thread.

**Comment by Donalcon** [comment]
Very interested to hear any responses on this too, is the default imgsz 640? can it be changed?
